R/shared-methods.R

```diff
@@ -37,7 +37,7 @@ check_country_available <- function(country = character(), level = 1,
   level <- match.arg(level, choices = c("1", "2"), several.ok = FALSE)
   tar_level <- paste0("level_", level, "_region")
 
-  if (is.null(region_class[[tar_level]])) {
+  if (is.null(region_class[["localise_regions"]][[tar_level]])) {
```

Comment:
I think we don't want to force them to add region codes that don't exist?

Reply:
This runs exactly the same as previous behaviour - target level detected by presence of field (now list) "level_x_region".

So even in the old behaviour I think you had to add a "level_x_region" field to get it working. 

If there is no localised name for a level, you would just have to set "level_x_region" = "level_x_region".